tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -92,35 +91,21 @@ class CreateNodegroupParams(TypedDict):
     nodegroup_role_arn: str
 
 
-class TestEksCreateClusterOperator(unittest.TestCase):
-    def setUp(self) -> None:
+class TestEksCreateClusterOperator:
```

Comment:
1. replace TestCase.subTest by parametrize tests
2. Rename method `nodegroup_setUp` to `nodegroup_setup`
3. Rename method `fargate_profile_setup ` to `fargate_profile_setup `